languages/javascript/keywords/let.md

```diff
@@ -1 +1,59 @@
 # `let`
+The `let` keyword declares and initializes a new variable. It can be optionally defined to a value.
```

Comment:
```suggestion

The `let` keyword declares and initializes a new variable. It can be optionally defined to a value.
```

Add newline

Reply:
Did you mean:

```suggestion

The `let` keyword declares a block scope local variable, optionally initializing it to a value.
```

Reply:
Does
```javascript
let foo;
```
not initialize the `foo` variable?

Reply:
It initialises it to `undefined`. The problem is this:

> It can be optionally defined to a value

```javascript
let foo = 'x'
```

Defines `foo`, and initialises (assigns for the first time) a string literal of `x`. It doesn't optionally define to a value. The wording is just slightly off :)